### DISCUSSION TITLE
Support Custom TTS integrations

### DISCUSSION THREAD

**Discussion by sheminasalam** [nimroddolev/chime_tts#27] [General]
Even with very limited days of use, this integration is proving useful. Appreciate the good work. Although I was wondering if this could be made to support custome tts integrations like [Microsoft edge TTS](https://github.com/hasscc/hass-edge-tts) which is a variation of the native tts but without using api key and all.

**Comment by nimroddolev** [comment]
Thank you 🙏 
The integration already supports Microsoft Edge TTS. To use it set the ‘tts_platform’ to ‘edge_tts’.
It will be one of the selectable options in an upcoming release.

**Reply by flinthamm** [reply]
I have no idea why this project doesn't seem to have more traction and I hope it does in the future, as more people find it. 

Just wanted to echo @sheminasalam comments, after only VERY limited use, this integration is already proving very useful - Thank you for your hard work :)
